Assets/Scripts/Interaction Scripts/ControllerState.cs

```diff
@@ -0,0 +1,195 @@
+﻿namespace ISAACS {
+
+    using System.Collections;
+    using System.Collections.Generic;
+	using VRTK;
+    using UnityEngine;
+
+	public class ControllerState : MonoBehaviour {
+
+		// Remember to set these controllers to actual GameObjects through the Unity GUI.
+		public GameObject LeftController;
```

Comment:
What is the difference between LeftController and controller_left? Same for the right ones. Please add documentation comment.